Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Debian oldoldstable, support ends in May 2018
+# we need this for Ruby 1.9.x at the moment
+FROM node:6.13.1-wheezy
+
+# TODO: base image
+USER root
+RUN useradd -ms /bin/bash -G www-data elife && \
+    chown elife:elife /srv && \
+    mkdir /srv/bin && \
+    chown elife:elife /srv/bin && \
+    mkdir -p /var/www && \
+    chown www-data:www-data /var/www
+ENV PATH=/srv/bin:${PATH}
+# end base image
+
+# potentially base image
+RUN npm install -g gulp-cli
+# end potentially base image
+
+RUN apt-get update && apt-get install -y \
+    bzip2 \
+    g++ \
+    make \
+    ruby \
+    ruby-dev 
+
+RUN gem install compass
```

Comment:
@davidcmoulton We really need to remove this since it's unsupported.

Reply:
Removes the Ruby constraint too.

Reply:
:man_shrugging: I replicated what this is doing in production now

Reply:
Agree. Ticket raised (ELPP-3514).